tests/testthat/test-int-latent_individual.R

```diff
@@ -1,10 +1,11 @@
 prep_obs <- as_latent_individual(sim_obs)
 prep_obs_gamma <- as_latent_individual(sim_obs_gamma)
 
-test_that("epidist.epidist_latent_individual Stan code compiles in the default case", { # nolint: line_length_linter.
+test_that("epidist.epidist_latent_individual Stan code has no syntax errors and compiles in the default case", { # nolint: line_length_linter.
   skip_on_cran()
   stancode <- epidist(data = prep_obs, fn = brms::make_stancode)
   mod <- cmdstan_model(stan_file = write_stan_file(stancode), compile = FALSE)
+  expect_true(mod$check_syntax())
```

Comment:
Are the comments here an issue : https://github.com/epinowcast/epinowcast/blob/d957b21af07c7708d8ae28fc248fffe39d453828/.github/workflows/check-cmdstan.yaml#L67

The concern would be that `brms` has pedantic problems at some point and this then breaks out testing.